views/dashboardEdit.handlebars

```diff
@@ -0,0 +1,24 @@
+<script src="/static/js/dashboardInfoForm.js"></script>
+
+<form method="post">
+    <div id="form-container">    
+        <label for="organization-name">Name</label><br>
+        <input type="text" id="organization-name" name="organization-name"><br>
+
+        <label for="organization-description">Description</label><br>
+        <input type="text" id="organization-description" name="organization-description"><br>
+
+        <label for="organization-address">Address</label><br>
+        <input type="text" id="organization-address" name="organization-address"><br>
+
+        <label for="organization-phone">Phone</label><br>
+        <input type="text" id="organization-phone" name="organization-phone"><br>
+
+        <label for="organization-website">Website</label><br>
+        <input type="text" id="organization-website" name="organization-website"><br>
+
+        <label for="organization-email">Email</label><br>
+        <input type="text" id="organization-email" name="organization-email"><br>
```

Comment:
There's a lot of common code between this and ```dasboardView.handlebars```. Could we make a partial for dashboard form?